test/onedrive.test.js

```diff
@@ -20,9 +20,7 @@ const OneDrive = require('../src/OneDrive.js');
 describe('OneDrive Tests', () => {
   it('throws when required parameters are not specified.', async () => {
     try {
-      // eslint-disable-next-line no-unused-vars
-      const drive = new OneDrive({});
-      assert.fail('This should not happen.');
+      assert.throws(new OneDrive({}));
```

Comment:
but now, you don't need the try/catch :-)

Reply:
Really? If I remove the try/catch, the test fails ;-)

Reply:
Right, I was missing the `() => new OneDrive({})` part, thanks, @stefan-guggisberg :-D